apps/web/src/features/safenet/components/new-safe/SafenetInfoCard/index.tsx

```diff
@@ -0,0 +1,48 @@
+import { AppRoutes } from '@/config/routes'
+import BellIcon from '@/public/images/safenet/bell.svg'
+import CloseIcon from '@mui/icons-material/Close'
+import { Box, Button, IconButton, Typography } from '@mui/material'
+import { useRouter } from 'next/router'
+import { useState } from 'react'
+import css from './styles.module.css'
+
+function SafenetInfoCard() {
+  const router = useRouter()
+  const [displayBanner, setDisplayBanner] = useState<boolean>(true)
```

Comment:
Usually we use the local storage to remember that a user closed a banner.

```suggestion
  const [displayBanner = true, setDisplayBanner] = useLocalStorage<boolean>("showSafenetBanner")
```

Reply:
The main Safenet banner and this card are the only access points to Safenet creation flow. This makes sense for the main banner in the dashboard, but it should be visible here every time the creation flow is started (with the dismiss option), otherwise there is no access point.